inc/saiswitch.h

```diff
@@ -472,6 +472,17 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_DEFAULT_VIRTUAL_ROUTER_ID,
 
     /**
+     * @brief Default .1Q Bridge ID
```

Comment:
by default we create a .1Q bridge and add all the switch port to the that bridge 
we need this attribute in order to get the default .1Q bridge object